### DISCUSSION TITLE
Crouton Problem

### DISCUSSION THREAD

**Discussion by roweenio** [dnschneid/crouton#4719] [Q&A] [answered]
Crashes on Unity Focal but not on Xenial

`(++) Log file: "/tmp/Xorg.crouton.1.log", Time: Fri Apr 22 11:01:38 2022
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.chromium.LibCrosService was not provided by any .service files
method return time=1650621699.391134 sender=:1.38 -> destination=:1.153 serial=1300 reply_serial=2
   boolean true
(II) modeset(0): Initializing kms color map for depth 24, 8 bpc.
amdgpu: os_same_file_description couldn't determine if two DRM fds reference the same file description.
If they do, bad things may happen!
crouton: version 1-20220111101058~master:a9eafaee
release: focal
architecture: amd64
xmethod: xorg
targets: unity
host: version 14526.89.0 (Official Build) stable-channel grunt 
kernel: Linux localhost 5.10.101-14962-gede4ccb2bf53 #1 SMP PREEMPT Wed Apr 13 19:47:05 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Running exit commands...
/usr/bin/xinit: connection to X server lost

waiting for X server to shut down Hangup
Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.chromium.LibCrosService was not provided by any .service files
method return time=1650621714.078490 sender=:1.38 -> destination=:1.158 serial=1631 reply_serial=2
   boolean true
(II) Server terminated successfully (0). Closing log file.

Unmounting /mnt/stateful_partition/crouton/chroots/focal...`

**Comment by TomTravis** [comment]
yes  i have added Focal to my efforts not working(xorg) just fine someone asked about this is Focal tries to use systemd at startup

**Reply by roweenio** [reply]
Thank you

**Comment by TomTravis** [comment, marked as answer]
xiwi  works Xorg does not on many configs nothing after buster debian and xenial on the default , the Xorg daemon does not yet start via multi user config systemd this is in work now. 
[systemd](https://github.com/systemd/systemd) is update years ahead of debian  highperformance servers use it, i have it installed and are working it , the system and user startups 
this installed to start [pid_1 ](https://mmonit.com/monit) clone and build.  Init.d is no longer used in the new systemd, chroot is still not configured this is not a simple install from crouton debootstrap i wish it was the chromebook is a low cost workstation

**Reply by roweenio** [reply]
I've tried this and it works like normal. No errors! Thank you

**Reply by TomTravis** [reply]
> I've tried this and it works like normal. No errors! Thank you

great you can be a vscode tester now i hope you have x86_x64 chromebook
![Screenshot at 2022-05-31 12-57-33](https://user-images.githubusercontent.com/46386261/171277023-1b63cf0e-e40b-4f16-8f8b-6cacdbbfcf5c.png)
let me know if you would like to test

**Comment by TomTravis** [comment]
![Screenshot at 2022-05-31 19-43-01](https://user-images.githubusercontent.com/46386261/171317765-e292ea0f-962a-4e5e-a6e5-f56ec1af150e.png)
